src/net_processing.cpp

```diff
@@ -2631,6 +2631,7 @@ void PeerManagerImpl::ProcessBlock(CNode& node, const std::shared_ptr<const CBlo
     m_chainman.ProcessNewBlock(block, force_processing, &new_block);
     if (new_block) {
         node.m_last_block_time = GetTime<std::chrono::seconds>();
```

Comment:
Please save a time and re-use it to avoid the slim chance of setting these to different values.

Reply:
Done.